### PR TITLE
Update issue templates to reduce the size of text and make issues fit better

### PR DESCRIPTION
**Pull request by jonespm**
Changed H2 to H3 to better fit in the issue.
Reduced the bug expected behavior of description text.

### CODE REVIEW

**Comment by jonespm** [comment]
I was looking at our issues and H2 just is so big for me. I noticed some other repos go to H3 and that just seems better. This is a pretty small change, but makes future issues more readable (to me). 

Old: 
image

New (Proposed): 
image

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
 name: Feature request
 about: Suggest an idea for this project
 title: ''
-labels: enhancement
+labels: ''
```
Comment (ssciolla) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: What's the reasoning behind changing this? I'm used to automatically add the `enhancement` label for features, which seems right. I also wonder if the empty string is the right way to specify `null` or empty list.

Comment (jonespm) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: Not sure, I edited in the in the UI, not sure why it removed this.

Comment (jonespm) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]:
```suggestion
labels: enhancement
```

**Pull request merged**